docs/architecture/sdk/index.md

```diff
@@ -0,0 +1,70 @@
+---
+sidebar_position: 3
+---
+
+# SDK Architecture
+
+Bitwarden provides a public Software Development Kit (SDK) for [Secrets Manager][sm] and an internal
+SDK for the Bitwarden [Password Manager][pm]. The SDK is written in Rust and provides bindings for
+multiple languages. The general end goal of the SDK is to own everything up to the presentational
+layers. This includes but is not limited to: API calls, data models, encryption, and business logic.
+
+<Bitwarden>We have collected a list of various learning resources on
+[Confluence](https://bitwarden.atlassian.net/wiki/spaces/DEV/pages/517898288/Rust+Learning+Resources).</Bitwarden>
+
+## Architecture
+
+The Bitwarden SDK is structured as a single [Git repository](https://github.com/bitwarden/sdk) with
+multiple internal crates. Review the repository README for more information about the different
+crates.
+
+We generally strive towards extracting features into separate crates to keep the core `bitwarden`
+crate as lean as possible. This has multiple benefits including:
+
+- Faster compile. Unchanged crates are not recompiled.
+- Promotes clear ownership of features.
```

Comment:
⛏️ With just two bullets I usually like to just make this one sentence, in the above paragraph.